activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb

```diff
@@ -141,6 +141,10 @@ def supports_datetime_with_precision?
         end
       end
 
+      def supports_virtual_columns?
+        !mariadb? && version >= "5.7.5"
```

Comment:
Thanks for the reply. Oh, I did not expect that combination.

One idea is not supporting virtual columns schema migrations at all. Well, it does not look friendliest but at least it is somewhat kind since users get to know the virtual column is not supported from the beginning, not at the timing when they are performing schema dumps after creating a bunch of migrations including virtual columns.

Another one is adding comments or documents to use SQL dump for virtual columns. I do not have other ideas yet.